imagetest/test_suites/storage_perf/storage_perf_utils.go

```diff
@@ -0,0 +1,128 @@
+package storageperf
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	vmName = "vm"
+	// iopsErrorMargin allows for a small difference between iops found in the test and the iops value listed in public documentation.
+	iopsErrorMargin = 0.97
+	// hyperdiskSize is used to determine which partition is the mounted hyperdisk.
+	hyperdiskSize = 100
+	bootdiskSize  = 10
```

Comment:
Boot disk size varies by OS. For EL based images, it is 20GB by default, and Windows uses 30GB. I would suggest making this either larger than the biggest public image (50GB should do it) or allowing the API to create the default size (which is whatever the image size is).

Reply:
changed to 50gb